lib/generators/doorkeeper/templates/migration.rb.erb

```diff
@@ -1,6 +1,6 @@
 class CreateDoorkeeperTables < ActiveRecord::Migration<%= migration_version %>
   def change
-    create_table :oauth_applications do |t|
+    create_table :oauth_applications, <%= primary_id_option %> do |t|
```

Comment:
Could not find in any of my Rails 5.2 projects this option, can you please share a link to PR or any article describing this?

Just run `rails g migration CreateUsers name:string` and got:

```ruby
class CreateUsers < ActiveRecord::Migration[5.2]
  def change
    create_table :users do |t|
      t.string :name
    end
  end
end
```